desktop/apps/react_example/index.js

```diff
@@ -0,0 +1,16 @@
+import 'babel-core/register'
+import express from 'express'
+import reloadable, { isDevelopment } from 'desktop/lib/reloadable'
+
+const app = module.exports = express()
+
+app.set('view engine', 'jade')
+app.set('views', `${__dirname}/components`)
+
+if (isDevelopment) {
+  reloadable(app, (req, res, next) => {
```

Comment:
When I get some time I'm going to figure out how to hoist this up into the main app framework so that one doesn't have to conditionally render things, but for now this should suffice